server/internal/types.proto

```diff
@@ -8,3 +8,10 @@ message RefreshToken {
   string refresh_id = 1;
   string token = 2;
 }
+
+// OfflineSessionID represents both the userID and connID required to 
+// uniquely identify an OfflineSession object
+message OfflineSessionID {
```

Comment:
nit: Can we just call this `IDTokenSubject` or something? That's actually what we're using this for.